2017/11/29/systemd-primer-from-the-trenches.html.md

```diff
@@ -0,0 +1,514 @@
+---
+author: Ian Neilsen
+title: "systemd: A primer from the trench's"
+tags: hosting, systemd, systemctl 
+---
+
+# systemctl - lets get back to basics
+
+''help me systemd you are my only hope''. If we start back at the beginning, sometimes going back to day dot
```

Comment:
Is "day dot" a turn of phrase like "day zero"? I'm not familiar with "day dot".

Reply:
change to day zero